internal/pkg/vm/internal/vmcontext/syscalls.go

```diff
@@ -26,8 +26,9 @@ type SyscallsImpl interface {
 	HashBlake2b(data []byte) [32]byte
 	ComputeUnsealedSectorCID(ctx context.Context, proof abi.RegisteredProof, pieces []abi.PieceInfo) (cid.Cid, error)
 	VerifySeal(ctx context.Context, info abi.SealVerifyInfo) error
-	VerifyPoSt(ctx context.Context, info abi.PoStVerifyInfo) error
-	VerifyConsensusFault(ctx context.Context, h1, h2, extra []byte, head block.TipSetKey, view SyscallsStateView, earliest abi.ChainEpoch) (*specsruntime.ConsensusFault, error)
+	VerifyWinningPoSt(ctx context.Context, info abi.WinningPoStVerifyInfo) error
```

Comment:
Either I'm missing something or specs actors remove this call cc @anorth

Reply:
*specs-actors should remove this call

Reply:
I did this speculatively. I'll remove it.